lib/kramdown/converter/syntax_highlighter/rouge.rb

```diff
@@ -17,17 +17,23 @@ module Rouge
 
       # Highlighting via Rouge is available if this constant is +true+.
       AVAILABLE = true
+
+      # Rouge::Formatters::HTMLLegacy is available on Rouge 2.0 or later
+      FORMATTER_CLASS = ::Rouge::Formatters::HTMLLegacy
     rescue LoadError, SyntaxError
       AVAILABLE = false  # :nodoc:
+    rescue NameError
+      # Fallbacks to Rouge 1.x formatter if Rouge::Formatters::HTMLLegacy is not available
+      FORMATTER_CLASS = ::Rouge::Formatters::HTML
     end
 
     def self.call(converter, text, lang, type, call_opts)
       opts = options(converter, type)
       call_opts[:default_lang] = opts[:default_lang]
       lexer = ::Rouge::Lexer.find_fancy(lang || opts[:default_lang], text)
       return nil if opts[:disable] || !lexer
-
-      formatter = (opts.fetch(:formatter, ::Rouge::Formatters::HTML)).new(opts)
+      opts[:css_class] ||= 'highlight' # For backward compatibility when using Rouge 2.0
```

Comment:
Althogh this line actually does almost nothing when using Rouge < 2.0, I think it is better than checking a formatter's class in the method.